### PR TITLE
Update remote_webdriver_client.en.md

### PR DESCRIPTION
**Pull request by marcus-rp**
**Thanks for contributing to the Selenium site and documentation!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://selenium.dev/documentation/en/contributing/) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Types of changes

- [ ] Change to the site (I am attaching a screenshot showing the before and after)
- [ ] Code example added (and I also added the example to all translated languages)
- [ ] Improved translation
- [ ] Added new translation (and I also added a notice to each document missing translation)

### Checklist

- [x] I have read the [**contributing**](https://selenium.dev/documentation/en/contributing/) document.
- [ ] I have used [hugo](https://gohugo.io/) to render the site/docs locally and I am sure it works.

### CODE REVIEW

File: docs_source_files/content/remote_webdriver/remote_webdriver_client.en.md [hunk 1]
```diff
@@ -162,7 +162,7 @@ var remote = require('selenium-webdriver/remote');
 driver.setFileDetector(new remote.FileDetector);  
   {{< / code-panel >}}
   {{< code-panel language="kotlin" >}}
-driver.setFileDetector(new LocalFileDetector())
+driver.fileDetector = LocalFileDetector()
```
Comment (Harsha509) [inline docs_source_files/content/remote_webdriver/remote_webdriver_client.en.md]: hi @marcus-rp,

Good catch.
Can you please update code for other translated pages too

Comment (marcus-rp) [reply docs_source_files/content/remote_webdriver/remote_webdriver_client.en.md]: how do I do it?

Comment (diemol) [reply docs_source_files/content/remote_webdriver/remote_webdriver_client.en.md]: In the same directory you can find more files with the same name but a different suffixes... Those are the translations. Please also change the code in those files.
Don’t forget to sign the CLA please.

**Comment by Harsha509** [comment]
hi @marcus-rp ,

Thanks for providing the PR.

As the PR gone stale now, I've updated the sample code in PR #160

**Comment by diemol** [comment]
Replaced by #160

**Pull request closed without merging**